x/tradeshield/keeper/pending_spot_order.go

```diff
@@ -291,9 +290,9 @@ func (k Keeper) ExecuteLimitBuyOrder(ctx sdk.Context, order types.SpotOrder) (*a
 		Sender:    order.OwnerAddress,
 		Recipient: order.OwnerAddress,
 		Amount:    order.OrderAmount,
```

Comment:
amount for this should be order.OrderAmount * Rate, right ?
because that's what we need to send for swap